mesheryctl/internal/cli/root/system/logs.go

```diff
@@ -182,6 +193,26 @@ mesheryctl system logs meshery-istio
 			}
 
 			log.Info("Starting Meshery logging...")
+
+			logChan := make(chan logChanStruct)
+			wg := &sync.WaitGroup{}
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
+				for {
+					value, ok := <-logChan
+					if ok {
+						for _, msg := range strings.Split(string(value.Message), "\n") {
+							logStr := fmt.Sprintf("%s\t|\t%s", value.PodName, msg)
```

Comment:
Are we sure that we're making this command only k8s exclusive? What about other platforms?

Reply:
@alphaX86 Yes. For docker that is already taken care